app/commands/mentor/discussion/finish_by_student.rb

```diff
@@ -84,6 +85,10 @@ def award_badge!
     AwardBadgeJob.perform_later(discussion.mentor, :mentor)
   end
 
+  def award_trophy!
```

Comment:
Small nit - you have `award_trophy` here and `award_trophies` in the one above. Maybe better to standardise on the plural everywhere (probably for `award_badges` too?)

Reply:
🤷